js/modules/cohortbuilder/components/CohortExpressionEditor.js

```diff
@@ -288,29 +288,31 @@ define(['knockout', 'jquery', '../options', '../CriteriaGroup', '../CriteriaType
 
 		self.removePrimaryCriteria = function (criteria) {
 			self.expression().PrimaryCriteria().CriteriaList.remove(criteria);
-		};
+		}
 		
 		self.removeCensoringCriteria = function (criteria) {
 			self.expression().CensoringCriteria.remove(criteria);	
-		};
+		}
 
 		self.addInclusionRule = function () {
 			var newInclusionRule = new InclusionRule(null, self.expression().ConceptSets);
 			self.expression().InclusionRules.push(newInclusionRule);
 			self.selectInclusionRule(newInclusionRule);
-		};
+		}
 
 		self.deleteInclusionRule = function (inclusionRule) {
 			self.selectedInclusionRule(null);
 			self.expression().InclusionRules.remove(inclusionRule);
-		};
+		}
 
 		self.copyInclusionRule = function (inclusionRule) {
 			var copiedRule = new InclusionRule(ko.toJS(inclusionRule), self.expression().ConceptSets);
 			copiedRule.name("Copy of: " + copiedRule.name());
 			self.expression().InclusionRules.push(copiedRule);
 			self.selectedInclusionRule(copiedRule);
-		};
+		}
+
+		self.addConceptSet = function (item) {}
```

Comment:
This function might be unnecessary, I noticed elsewhere in the code that there's an event handler for the .addconceptset selector, but I'm wondering if that selector should be removed vs. adding a empty addConceptSet function...was there an error you were receiving where you needed to add this event handler?